_meeting-notes/10-10-2019.md

```diff
@@ -0,0 +1,27 @@
+# Sync Oct 10, 2019
+
+## Attendees
+
+- David Dias
+- Jim Pick
+
+## Notes
+
+- AI(David): create PR to list the current team and roles 
+  - Raul -> Architect 
+  - Jim
+  - David
+- AI(David): Review the 1st video demo by Jim
+- AI(Jim): Open issues that cover the following topics/questions to get answers/shared understanding
+  - 1) What are the main differences (scope, features, ) between TestGround, TestLab, ipfs/benchmarks?
+  - 2) ElasticSearch Review
+    - Write up of the learning of the multiple ElasticSearch hosted versions vs. hosting ourselves. Pros & Cons
+    - Questions
+      - What is the best one to use?
+      - 
+  - 3) Graphana vs Kibana
+    - List the degrees of freedom of each. Concerns about Kibana
+  - 4) Why implement sidecar vs. using filebeat
+  - **Update:** No need to create this one. Sidecar exists to collect things beyond logs (such as assets, pprofs and more)
+- AI(Jim): Create a 3 minute of the DHT TestPlan results showing up on the Kibana dashboard (the demo that was supposed to show at the IPFS All Hands)
```

Comment:
Created the card for this one https://trello.com/c/S5R59r9D/58-create-a-3-minute-of-the-dht-testplan-results-showing-up-on-the-kibana-dashboard

Reply:
Here's a 23 second video: https://ipfs.io/ipfs/Qmahm2x4UGqGgkeVhijTF1SvmekhiphfLgK2uQxTU95y8P/testground-filebeat.mp4